server.js

```diff
@@ -0,0 +1,72 @@
+var express = require("express"); //require is a method that comes with node, runs the file it is given.
+                                  //require returns the value assigned by the file to module.exports (if any)
+var app = express(); //app is now something we can use dot notation with, to use all the methods that come with express
+                     //server.js will be responsible for taking orders (incoming traffic) and dispensing things
+
+var port = process.env.PORT || 3000; //tells Express to listen for http requests on port 3000
+                                     //(unless we've set a different port in an environmental variable)
+
+var bodyparser = require("body-parser"); //this node module will automatically parse .json for us
+app.use(bodyparser.json()); //These two lines tell the express app to use the bodyparser module...
+app.use(bodyparser.urlencoded({ extended: true })); //...and give it some needed settings.
+  //(it needs this to actually be able to decode the data coming in on a request.)
+
+var piglatinify = require("./lib/piglatinify.js"); //it's happily in its own file. it's free now. freeeeee
+var quotes = require("./lib/quotes.js");
+var thoughts = require("./lib/thoughts.js");
+var buttons = require("./lib/newbuttons.js");
+var yesorno = require("./lib/yesorno.js");
+//modular design patterns are cool!
```

Comment:
yes, yes they are. Your file is now super-clean and only has server-related functionality in it. Woot!